scripts/service_provider.py

```diff
@@ -0,0 +1,35 @@
+import csv
+
+websites = {}
+
+# 12, 13, 16
+with open('../data/service_providers/service_providers_raw.csv', newline='', encoding="ISO-8859-1") as file:
```

Comment:
```suggestion
SERVICE_PROVIDERS_FILE = '../data/service_providers/service_providers_raw.csv'
with open(SERVICE_PROVIDERS_FILE, newline='', encoding="ISO-8859-1") as file:
```

Reply:
there may be a better name than `SERVICE_PROVIDERS_FILE` but i think we should break out the constants from the code